health/health_test.go

```diff
@@ -49,3 +52,107 @@ func TestNilMethodsDontCrash(t *testing.T) {
 		rv.Method(i).Call(args)
 	}
 }
+
+func TestSetUnhealthyWithDuplicateThenHealthyAgain(t *testing.T) {
+	newTracker := Tracker{}
```

Comment:
`var ht Tracker` ?

"newTracker" is a long name and very specific (what's new about it? how long does it stay new? is there an old one?)

Reply:
done